util.go

```diff
@@ -0,0 +1,32 @@
+package inflight
+
+import (
+	"math"
+)
+
+func min(a, b float64) float64 {
+	if a <= b {
+		return a
+	}
+	return b
+}
+
+func ACS(pl PriorityBand, queues []*Queue) int {
```

Comment:
Why is this function here?  The KEP says that each priority level is directly configured with an ACS value.